### PR TITLE
Missing migrations

### PR DESCRIPTION
**Pull request by rowanseymour**
Adds all the missing migrations (typically updating `choices` or `help_text`) and adds check to Travis for new missing migrations.

### CODE REVIEW

File: temba/auth_tweaks/models.py [hunk 1]
```diff
@@ -1,6 +0,0 @@
-from django.contrib.auth.models import User
-
-field = User._meta.get_field('email')
```
Comment (rowanseymour) [inline temba/auth_tweaks/models.py]: These tweaks aren't needed and don't actually update the database (we must have done that manually?). The other downside of these tweaks is that they require a migration in the auth app. Apparently all will be fixed in Django 1.9.? or 1.10.. https://code.djangoproject.com/ticket/20846

File: .travis.yml [hunk 2]
```diff
@@ -28,6 +28,7 @@ script:
 - node_modules/karma/bin/karma start karma.conf.coffee --single-run --browsers PhantomJS
 - coverage run manage.py test --noinput --verbosity=2
 - flake8
+- python manage.py makemigrations --dry-run | grep 'No changes detected' || (echo 'There are changes which require migrations.' && exit 1)
```
Comment (rowanseymour) [inline .travis.yml]: Fails the build if you didn't remember to generate migrations for model changes

**Comment by nicpottier** [comment]
Nice. Great to have a way of making sure these don't drift with new choices. Somewhat nervous that this might cause a db lock as we ship it, but we'll see. (since we are seem to be always running auto vacuums table schema changes lock unless we kill the vaccuum manually which is a pain)

File: temba/auth_tweaks/models.py (hunk 1 above)
Comment (nicpottier) [inline temba/auth_tweaks/models.py]: They did at some point, but maybe things have changed. Having the manual migration seems just fine though.

Comment (rowanseymour) [inline temba/auth_tweaks/models.py]: Just to clarify my comment above - it's the original tweaks that I've removed that generate a migration for the auth app which we can't save anywhere and would obviously conflict with any official migrations for that app.

**Pull request merged**